CoCSharp.Server/Handlers/AllianceMessageHandlers.cs

```diff
@@ -109,6 +109,13 @@ private static void HandleChangeAllianceSettingMessage(CoCServer server, AvatarC
             }
 
             clan.Description = edAlliance.Description;
+            clan.Badge = edAlliance.Badge;
+            clan.InviteType = edAlliance.Type;
```

Comment:
It would be great if you could follow the naming style and names, to avoid any confusion.